deployments/base/weth/configuration.json

```diff
@@ -5,15 +5,15 @@
   "baseTokenAddress": "0x4200000000000000000000000000000000000006",
   "borrowMin": "1e0",
   "pauseGuardian": "0x3cb4653f3b45f448d9100b118b75a1503281d2ee",
-  "storeFrontPriceFactor": 0.5,
+  "storeFrontPriceFactor": 1,
```

Comment:
interesting; so the protocol would make nothing for a liquidation?

Reply:
Yeah, based on the recommendations in this post: https://www.comp.xyz/t/gauntlet-recommendations-ethereum-compound-v3-eth-risk-parameter-ir-curve-incentive-changes-8-4-23/4565